src/Juvix/Backends/ArithmeticCircuit/Proof.hs

```diff
@@ -0,0 +1,43 @@
+module Proof (proof) where
```

Comment:
It seems this file does not really compile?

The project builds, as this isn't added to the yaml file

Reply:
Oh damn, I thought I had deleted it. I replaced it with `ZKP.hs`.